README.md

```diff
@@ -9,9 +9,9 @@ See [Tracing QUIC events](#tracing-quic-events) for how.
 See [requirements](#requirements) for build prerequisites.
 
 ```
-$ cmake .
-$ make
-$ sudo make install
+$ cmake -H. -Bbuild
+$ make -Cbuild
+$ sudo make -Cbuild install
```

Comment:
Not directly related to Ubuntu 20.04, but I strongly recommend out-of-source builds for those which use `cmake`.

Reply:
Sure, I don't have any objections to prompting developers to build out-of-source.